bin/i18n/sync-in.rb

```diff
@@ -376,6 +377,18 @@ def localize_shared_functions
   end
 end
 
+def localize_course_offerings
+  puts "Preparing course offerings"
+
+  hash = {}
+  CourseOffering.all.sort.each do |co|
+    hash[co.key] = co.display_name
+  end
+  File.open(File.join(I18N_SOURCE_DIR, "dashboard/course_offerings.yml"), "w+") do |f|
```

Comment:
Instead of yml, could we make this a JSON instead? I know the pattern we have mostly followed in the past is creating YML files, but I would like us to move towards using JSON files instead since they are easier for use to maintain.